src/components/search/doc-search.js

```diff
@@ -0,0 +1,11 @@
+import "@docsearch/css";
+import docsearch from "@docsearch/js";
+
+export function init(container) {
+    docsearch({
+        container,
+        indexName: "skin",
+        appId: "4AG9MNELVM",
+        apiKey: "9b7a33ff8f0985b0f2b66e90c020b459",
```

Comment:
Are these okay go in here? Should we be using github secrets?
Just double check so that we don't introduce a security loophole.

Reply:
The DocSearch documentation is not clear (I know, irony, right?). This was the only clear implementation example I found. Marko team is using it the same way. Maybe since the actual "work" is being done from the account UI after login, it merely needs to tie into the app? idk